src/otx/__init__.py

```diff
@@ -5,8 +5,14 @@
 
 __version__ = "2.2.0rc0"
 
+import os
+from pathlib import Path
+
 from otx.core.types import *  # noqa: F403
 
+# Pretrained weight is saved into ~/.cache/torch/hub/checkpoints
+os.environ["XDG_CACHE_HOME"] = str(Path.home() / ".cache" / "torch" / "hub" / "checkpoints")
```

Comment:
do we need to make a directory if it doesn't exist?

Reply:
When I deleted and restarted it, it created the folder on its own.